utils/nctl/sh/scenarios/common/itst.sh

```diff
@@ -46,9 +46,43 @@ function log_step() {
 }
 
 function do_await_genesis_era_to_complete() {
-    log_step "awaiting genesis era to complete"
+    local LOG_STEP=${1:-'true'}
+    local TIMEOUT=${2:-'120'}
```

Comment:
That's 2 minutes, correct? How long are eras in these tests usually by default?

Reply:
`2mins` yes. Locally it was consistently completing in about `60s`. Gave it a little extra buffer.

`era_duration` is `41s` for most. But thanks for the question, reminds me of two more tests I need to extend the timeout on!